### DISCUSSION TITLE
Slick Slider do not running in github

### DISCUSSION THREAD

**Discussion by Bayraktar13** [community/community#56714] [Discussions]
### Select Topic Area

General

### Body

I have connected "Slick Slider". When I checking localy, there is no any error or mistakes, but when I pushing to github, the slick slider is not running. 
What should I do to fix this problem?
Please assist!

**Comment by Daeraxa** [comment]
What is slick slider and what relationship does it have to GitHub Discussions?

**Reply by Kashumy** [reply]
what ? 🤨

**Reply by Daeraxa** [reply]
What?
